### PR TITLE
Move Tippy CSS to our bundle 🧵

### PR DESCRIPTION
**Pull request by lluia**
## Summary 

We were importing **Tippy CSS** through a [CSS import on `<GlobalStyles />`](https://github.com/zopaUK/react-components/blob/master/src/components/styles/GlobalStyles.tsx).

@moshie and @endymion1818 fairly pointed out that it increased a bit the _HTTP handshake_ on the consumers not using tooltips. 

The main issue though is that [`styled-components@5` does not support anymore CSS `@import`](https://github.com/styled-components/styled-components/issues/2995).

There's work planned on [moving to just PopperJS](https://github.com/zopaUK/react-components/issues/245).
 

## Related

https://github.com/zopaUK/react-components/pull/165
https://github.com/zopaUK/react-components/issues/234

https://github.com/styled-components/styled-components/issues/2995

### CODE REVIEW

**Comment by lluia** [comment]
I'm thinking to base this against `master` instead given it could go out now 🤔

File: package.json [hunk 1]
```diff
@@ -57,7 +56,7 @@
     "@types/jest": "^24.9.0",
     "@types/jest-axe": "^3.2.1",
     "@types/node": "12.12.27",
-    "@types/react": "^16.9.22",
+    "@types/react": "16.9.19",
```
Comment (lluia) [inline package.json]: I'm still seeing type errors related to this package, need to investigate further 🧐

File: etc/rollup/config.js [hunk 2]
```diff
@@ -6,7 +6,7 @@ import url from 'rollup-plugin-url';
 import babel from 'rollup-plugin-babel';
 import commonjs from 'rollup-plugin-commonjs';
 import { terser } from 'rollup-plugin-terser';
-import json from '@rollup/plugin-json';
+import css from 'rollup-plugin-css-only';
```
Comment (poteirard) [inline etc/rollup/config.js]: Have you tested the compile command? I think it will not work since you still have `json()` in line 34.

Comment (lluia) [reply etc/rollup/config.js]: ugh no 🤦🏻‍♂️ , on it 👍

Comment (lluia) [reply etc/rollup/config.js]: Hmm seems fine after removing it, however, I see this warning 🧐
```
$ rollup --config ./etc/rollup/config.js

/Users/lluis.agusti/Code/zopa-react-components/src/index.ts → cjs, es...
(!) Mixing named and default exports
https://rollupjs.org/guide/en/#output-exports
The following entry modules are using named and default exports together:
src/components/atoms/Dropdown/Dropdown.tsx
node_modules/babel-preset-react-app/node_modules/@babel/runtime/regenerator/index.js

Consumers of your bundle will have to use chunk['default'] to access their default export, which may not be what you want. Use `output.
exports: 'named'` to disable this warning
[!] (plugin css) Error: ENOENT: no such file or directory, mkdir
Error: ENOENT: no such file or directory, mkdir
```

**Comment by lluia** [comment]
I'm going to change a bit the approach on this PR 💭

I think I'll follow the same API as with [`<Modal.Styles />`](https://zopauk.github.io/react-components/#/Components/Molecules/Modal) and force clients to declare explicitly `<Tooltip.Styles />` on their **top-level component**.

File: etc/rollup/config.js (hunk 2 above)
Comment (poteirard) [reply etc/rollup/config.js]: This happens also in master so it can be addressed in another PR

**Pull request merged**